README.tfdoc.hcl

```diff
@@ -95,63 +90,143 @@ section {
     section {
       title = "Main Resource Configuration"
 
-      # please add main resource variables here
+      variable "project" {
+        type        = string
+        description = <<-END
+            The ID of the project in which the resources belong. If it is not set, the provider project is used.
+          END
+      }
 
-      # TODO: remove examples
+      variable "network" {
+        required    = true
+        type        = string
+        description = <<-END
+            The VPC network the subnets belong to. Only networks that are in the distributed mode can have subnetworks.
```

Comment:
Can we make this more clear? What's required? The `id` or the `name` of the network?